src/Misc/externals.sh

```diff
@@ -189,3 +189,8 @@ if [[ "$PACKAGERUNTIME" == "linux-arm" ]]; then
     acquireExternalTool "$NODE_URL/v${NODE16_VERSION}/node-v${NODE16_VERSION}-linux-armv7l.tar.gz" node16 fix_nested_dir
     acquireExternalTool "$NODE_URL/v${NODE20_VERSION}/node-v${NODE20_VERSION}-linux-armv7l.tar.gz" node20 fix_nested_dir
 fi
+
+if [[ "$PACKAGERUNTIME" == "linux-loongarch64" ]]; then
+    acquireExternalTool "http://ftp.loongnix.cn/nodejs/LoongArch/dist/v16.17.1/node-v16.17.1-linux-loong64.tar.gz" node16 fix_nested_dir
```

Comment:
The loongarch64 nodejs is not distributed by nodejs community.
And the versions of loongarch64 are not continuously released at present, I cannot keep the version the same as x64 or arm64.